src/main/java/uk/gov/laa/ccms/caab/bean/validators/client/ClientContactDetailsValidator.java

```diff
@@ -79,6 +79,7 @@ private void validateTelephoneField(
    */
   private void validateEmailField(ClientDetails clientDetails, Errors errors) {
     if (!StringUtils.hasText(clientDetails.getEmailAddress())
+        && clientDetails.getCorrespondenceMethod() != null
```

Comment:
Pedantic I know, but if you reverse it then you don't need the null check.
CORRESPONDENCE_EMAIL.equalsIgnoreCase(clientDetails.getCorrespondenceMethod())